Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -0,0 +1,105 @@
+              --namespace $CILIUM_NAMESPACE \\
+              --set global.hubble.enabled=true \\
+              --set global.hubble.metrics.enabled="{dns,drop,tcp,flow,port-distribution,icmp,http}"
+
+     .. group-tab:: Distributed Mode (beta)
+
+
+        In **distributed mode (beta)**, Hubble listens on a TCP port on the host network.
+        This allows :ref:`hubble_relay` to communicate with all the Hubble instances in
+        the cluster. Hubble CLI and Hubble UI in turn connect to Hubble Relay to provide
+        cluster-wide networking visibility.
+
+        .. warning::
+
+           In Distributed mode, Hubble runs a gRPC service over plain-text HTTP on the host
+           network without any authentication/authorization. The main consequence is that
+           anybody who can reach the Hubble gRPC service can obtain all the networking
+           metadata from the host. It is therefore **strongly discouraged** to enable
+           distributed mode in a production environment.
+
+        .. parsed-literal::
+
+           helm upgrade cilium |CHART_RELEASE| \\
+              --namespace $CILIUM_NAMESPACE \\
+              --set global.hubble.enabled=true \\
+              --set global.hubble.listenAddress=":4244" \\
+              --set global.hubble.metrics.enabled="{dns,drop,tcp,flow,port-distribution,icmp,http}" \\
+              --set global.hubble.relay.enabled=true \\
+              --set global.hubble.ui.enabled=true
+
+* Restart the Cilium daemonset to allow Cilium agent to pick up the ConfigMap changes:
+
+  .. parsed-literal::
+
+      kubectl rollout restart -n $CILIUM_NAMESPACE ds/cilium
+
+* To pick one Cilium instance and validate that Hubble is properly configured to listen on
+  a UNIX domain socket:
+
+  .. parsed-literal::
+
+      kubectl exec -n $CILIUM_NAMESPACE -t ds/cilium -- hubble observe
+
+* **(Distributed mode only)** To validate that Hubble Relay is running, download the ``hubble``
+  CLI from `the GitHub release page <https://github.com/cilium/hubble/releases>`_ and install it
+  in a directory listed in the ``PATH`` environment variable. Then, set up a port forwarding for
+  ``hubble-relay`` service and run ``hubble observe`` command:
+
+  .. parsed-literal::
+
+      kubectl port-forward -n $CILIUM_NAMESPACE svc/hubble-relay 4245:80
+      hubble observe --server localhost:4245
+
+  For convenience, you may set and export the ``HUBBLE_DEFAULT_SOCKET_PATH``
+  environment variable:
+
+  .. code:: bash
+
+    $ export HUBBLE_DEFAULT_SOCKET_PATH=localhost:4245
+
+  This will allow you to use ``hubbble status`` and ``hubble observe`` commands
+  without having to specify the server address via the ``--server`` flag.
+
+* **(Distributed mode only)** To validate that Hubble UI is properly configured, set up a port forwarding for
+  ``hubble-ui`` service:
+
+  .. parsed-literal::
+
+      kubectl port-forward -n $CILIUM_NAMESPACE svc/hubble-ui 12000
+
+  and then open http://localhost:12000/.
```

Comment:
Should this validation section also be split by local / distributed modes?

Reply:
i kind of wanted to keep these parts visible by default so that people see what additional capabilities they get from the distributed mode.